### PR TITLE
Devops days Seattle 2016 update part 2

### PR DESCRIPTION
**Pull request by jeremyd506**
Merged remote master and resolved merge conflicts. Think i got everything if not please let me know.

Thanks.

### CODE REVIEW

File: site/content/events/2016-seattle/index.html [hunk 1]
```diff
@@ -11,8 +11,7 @@
 <center>
   <table><tr><td>
         <center>
-        <!-- img border=0 width="160" height="240" src="./yourlogohere.png">
--->
+        <!-- <img border=0 width="160" height="240" src="./yourlogohere.png"> -->
```
Comment (bridgetkromhout) [inline site/content/events/2016-seattle/index.html]: I think you want this to be  ?

Comment (bridgetkromhout) [inline site/content/events/2016-seattle/index.html]: I think you want this to be: ``````

Comment (jeremyd506) [inline site/content/events/2016-seattle/index.html]: we are having an artist draw up some logos for us to use. So i was just going to leave the logo blank until we get that done in a few weeks.

Thanks.

Comment (bridgetkromhout) [inline site/content/events/2016-seattle/index.html]: Okay, good deal. Thanks.

**Pull request merged**